discord/abc.py

```diff
@@ -340,7 +342,7 @@ def overwrites_for(self, obj):
                 deny = Permissions(overwrite.deny)
                 return PermissionOverwrite.from_pair(allow, deny)
 
-        return PermissionOverwrite()
+        return None
```

Comment:
`return None` is redundant, a function will implicitly return None if it returns nothing at all.

Reply:
Interesting, didn't know that. Shouldn't it be left for clarity tho? I kinda want to leave it there to explicitly state that if no overwrites match, I want to return None instead of the usual overwrite that'd be returned two lines higher.